package com.catshi.core.exceptions

class TeamNameAlreadyExistsException: Throwable() {
}